elk/training/ccs_reporter.py

```diff
@@ -88,6 +88,8 @@ def __init__(
         num_variants: int = 1,
     ):
         super().__init__()
+        self._is_training = True
```

Comment:
🟡 I think this approach basically works but perhaps the wrapper approach that the EigenFitter/Reporter uses is cleaner

Reply:
In fact maybe we should just do that and make `CcsReporter.fit()` return a `Reporter`. Maybe need to have add a param that disables the eraser.

Reply:
Yeah, I removed the _is.training again. 

> In fact maybe we should just do that and make CcsReporter.fit() return a Reporter. Maybe need to have add a param that disables the eraser.

We can take a look at that. Could be maybe a new pull request or the same...